frontend/app/views/index.scala.html

```diff
@@ -8,7 +8,7 @@
 @import views.support.NavAnchor
 @import views.support.Asset
 
-@main("Join", pageInfo) {
+@main("", pageInfo) {
```

Comment:
Ideally I'd make title an `Option` in `main.scala.html`, but this is the minimum change needed to get this out.